docs/hardware/4-nrf91/2-quickstart/3-flash-firmware/2-bootloader.md

```diff
@@ -0,0 +1,123 @@
+---
+id: bootloader
+title: Flash via USB Bootloader
+---
+
+## Installing Tools
+
+Flashing via USB bootloader requires the `newtmgr` tool.
+
+### Installing Newt Manager (for bootloading)
+
+We recommend you use [Jared Wolff's fork of newtmgr](https://github.com/circuitdojo/mynewt-newtmgr) which includes customization for higher-speed transfers on MCUboot implementations that support it. This tool depends on the Go language so we will also install that.
+
+1. Install the Go Language
+
+    :::note
+    You can install the Go language with `sudo apt install golang-go`, but some package managers have older versions. The following process will install the newest version.
+    :::
+
+    Visit [the Go language download page](https://go.dev/dl/) and download the latest version. Install the package:
+
+    ```bash
+    cd ~
+    tar -xvf go1.17.6.linux-amd64.tar.gz --directory ~/.
+    nano ~/.profile
```

Comment:
Could add message about it possibly being `~/.bashrc` to the same effect (if using Bash)

Reply:
When looking into this it seems the `.profile` file is preferred for this type of path manipulation:
https://unix.stackexchange.com/questions/476593/when-should-i-use-bashrc-and-when-profile